php/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN apk add --no-cache --virtual .build-deps \
 RUN adduser -u 1001 -D php
 USER php
 COPY --from=symfony-cli /tmp/symfony /usr/local/bin/symfony
+RUN mkdir -p ~/.symfony
```

Comment:
I dream of no black magic from symfony cli one day :mage: :black_circle:

Reply:
I dream of symfony cli removal

Reply:
I dream of icecream :ice_cream: